### ISSUE TITLE
Unable to extract Animation ID

### ISSUE THREAD

**Issue by Haaris12345** [evaera/roblox-animation-transfer#23]
Everytime I try to run the generate list command, it always says "unable to extract animation id". Yes, I've set the ROBLOSECURITY in command prompt with "set "ROBLOSECURITY=blablabla" but that doesn't work. Not setting the ROBLOSECURITY doesn't work either as it gives a whole different error telling me "could not log in with provided cookie" or something along those lines. I think it's worth noting that I've used the animation transfer tool in the past and it has worked, but it's been a couple of months since I've done that.
![aff](https://user-images.githubusercontent.com/115205176/194467421-55cff9d6-fd34-4d5d-9560-a5bcec9eccf3.PNG)

**Comment by Haaris12345** [comment]
help

**Comment by WorldPosition** [comment]
getting this error too

**Comment by Haaris12345** [comment]
> getting this error too

think she abandoned the project so there's no hope left 😭😭😭

**Comment by kvfp** [comment]
I'm experiencing this same problem. I've tried a couple things to get around it to no avail:
- Disabling 2FA for my account (in case it was a security issue)
- Manually setting my cookie as a variable
- Manually entering my cookie with each command

I can confirm though that I was still able to quickly re-upload animations using this tool given I obtained the animation ids another way. I think the root of the issue might be that the API used to retrieve the animation list was recently deprecated? 
DevForum post: [Sunsetting api.roblox.com](https://devforum.roblox.com/t/action-needed-sunsetting-apirobloxcom/1796897)

Currently searching for alternatives!
